src/sentry/incidents/logic.py

```diff
@@ -553,6 +562,19 @@ def create_alert_rule(
 
     resolution = get_alert_resolution(time_window, organization)
 
+    if (sensitivity or seasonality) and not (sensitivity and seasonality):
```

Comment:
Can we stick this behind the `detection_type` check?
having these set only makes sense for the dynamic detection type right?